src/registry.rs

```diff
@@ -3594,6 +3594,7 @@ impl CheckKind {
                 | CheckKind::UselessImportAlias
                 | CheckKind::UselessMetaclassType
                 | CheckKind::UselessObjectInheritance(..)
+                | CheckKind::YodaConditions(..)
```

Comment:
@PedramNavid - This was the missing piece. There's a separate `fixable` "list" that marks a check as fixable in the README. (Autofix will still "work" without this, but it won't appear in the README, on the CLI, etc.)